tests/normalization_test.py

```diff
@@ -57,6 +58,42 @@ def test_solid_angle_rectangular_keeps_relevant_masks():
     assert 'mask2' not in solid_angle.masks
 
 
+@pytest.mark.skipif(
+    'SANS2D00063091.nxs' not in os.listdir(), reason='Needs data to recreate reference'
+)
+@pytest.mark.filterwarnings("ignore:.*")
+def test_recreate_sans2d_solidangle_from_mantid_file():
+    simpleapi = pytest.importorskip("mantid.simpleapi")
+    scippneutron = pytest.importorskip("scippneutron")
+
+    ws = simpleapi.Load('SANS2D00063091.nxs')
+    radius = 0.00405 * sc.Unit('m')
+    length = 0.002033984375 * sc.Unit('m')
+
+    simpleapi.SetInstrumentParameter(
+        ws,
+        ParameterName='x-pixel-size',
+        ParameterType='Number',
+        Value=str((2 * radius).to(unit='mm').value),
+    )
+    simpleapi.SetInstrumentParameter(
+        ws,
+        ParameterName='y-pixel-size',
+        ParameterType='Number',
+        Value=str(length.to(unit='mm').value),
+    )
+
+    outWs = simpleapi.SolidAngle(ws, method='HorizontalTube')
+    da = scippneutron.from_mantid(outWs)['data']['spectrum', :120000:100]
+    fname = os.path.join(
+        os.path.dirname(__file__), 'SANS2D00063091.SolidAngle_from_mantid.hdf5'
+    )
+    da_old = sc.io.load_hdf5(filename=fname)
+    assert sc.identical(da, da_old)
```

Comment:
I suppose it depends on your point of view. Given the history of how you arrived at this point it makes sense to compare the Mantid result to a reference file.

But I feel that in general comparing "result of software A" to "result of software B" is a "better" test. It is easier to understand out of context, and is less likely to break. For example, what if we later make some changes to the code in `esssans`, change how the test of solid angle works.... but forget that we still have the old reference file. You have now a test that claims that everything is fine, but in fact it is not! This problem would not arise if you directly compared the results of the two implementations, without relying on some files that were written at some point in the past.

Reply:
I see what you mean. But if we compare with mantid directly instead of comparing with the reference file then we will only be able to run the test locally *if* mantid and scippneutron is installed in the same environment. I think that test will be run very rarely. With the reference file the test can be run in CI, making it much more likely we catch a regression.

How about this: In the test that recreates the mantid file we compare the results from both softwares directly and with the reference file, if any of them disagrees an error is reported. Maybe that was what you had in mind already?

Reply:
What I had in mind was:
1. You existing test, comparing the esssans result to the reference file.
2. A test that compares the esssans result to the Mantid result.

If Mantid is missing, the second test will not run.